src/Expr.h

```diff
@@ -1672,6 +1673,25 @@ class CoerceFromAnyExpr : public UnaryExpr {
     ExprPtr Duplicate() override;
 };
 
+/**
```

Comment:
Nit, not the usual comment style used in headers

Reply:
I actually think this is fine, from my inerpretation of our [style guide](https://github.com/zeek/zeek/wiki/Coding-Style-and-Conventions), this should be the preferred style for something like that. We are not very consistent about it yet.

To add to this - at least for newly added functionality, I think it would be neat to add documentation for the invidual functions. I am not sure it makes entirely much sense for this specific case, so I am leaving that up to you :)